conan/tools/android/utils.py

```diff
@@ -0,0 +1,25 @@
+from conan.errors import ConanException
+
+
+def android_abi(conanfile, context="host"):
+    """
+    Returns Android-NDK ABI
+    :param conanfile: ConanFile instance
+    :param context: either "host", "build" or "target"
+    :return: Android-NDK ABI
+    """
+    if context not in ["host", "build", "target"]:
+        raise ConanException("context argument must be either 'host', 'build' or 'target'")
+
+    settings = getattr(conanfile, f"settings_{context}", conanfile.settings)
+    arch = settings.get_safe("arch")
+    # https://cmake.org/cmake/help/latest/variable/CMAKE_ANDROID_ARCH_ABI.html
+    return {
+        "armv5el": "armeabi",
+        "armv5hf": "armeabi",
+        "armv5": "armeabi",
+        "armv6": "armeabi-v6",
+        "armv7": "armeabi-v7a",
+        "armv7hf": "armeabi-v7a",
+        "armv8": "arm64-v8a",
```

Comment:
we're not handling x86 and MIPS variants here. at least x86 is popular for running Simulator builds (plus, there are real x86 boards with Android). MIPS was used by previous (old) NDK releases, but I would add for completeness.
(also I've heard Android is adding RISC-V now, but I don't know ABIs right now)

Reply:
arch in profile and ndk abi are identical for x86 and mips.

Reply:
it should be tested in unit test

Reply:
oh, good catch! sorry for the noise :)

Reply:
No it's fine, I've extended archs tested in the unit test.